src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1391,12 +1389,13 @@ public void Add(Double summand)
         }
     }
 
-    public sealed class SdcaBinaryTrainer : SdcaTrainerBase<SdcaBinaryTrainer.Arguments, BinaryPredictionTransformer<TScalarPredictor>, TScalarPredictor>
+    public sealed class SdcaBinaryTrainer : SdcaTrainerBase<SdcaBinaryTrainer.Options, BinaryPredictionTransformer<TScalarPredictor>, TScalarPredictor>
     {
         public const string LoadNameValue = "SDCA";
```

Comment:
>public [](start = 8, length = 6)

all those strings should be internal..

Reply:
is there some separate issue for it that I can link to this comment ? 

I would like to keep the PRs for public API  focused on API  rather than fixing issues across the codebase :)

---
In reply to: [248771027](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248771027) [](ancestors = 248771027)